Library/Homebrew/extend/os/mac/requirements/x11_requirement.rb

```diff
@@ -0,0 +1,49 @@
+require "requirement"
+
+Object.send(:remove_const, :X11Requirement)
```

Comment:
This is a bit weird.

Reply:
This line removes the class `X11Requirement` so that `extend/os/mac/requirements/x11_requirement.rb` can redefine it. Creating a new class with a new name like `XQuartzRequirement` and then assigning it with `X11Requirement = XQuartzRequirement` causes a test to fail because `be_an_instance_of(X11Requirement)` is false. We could make `X11Requirement` be a base class, and have `XQuartzRequirement` inherit from it. Alternatively, we could remove this test. Your thoughts?

Reply:
Never mind. I replaced it with `X11Requirement = XQuartzRequirement`.